trailblazer/cli/core.py

```diff
@@ -74,14 +74,15 @@ def log_cmd(context, sampleinfo, sacct, quiet, config):
 @click.option('-p', '--priority', type=click.Choice(['low', 'normal', 'high']), default='normal')
 @click.option('-d', '--dryrun', is_flag=True, help='only generate SBATCH scripts')
 @click.option('--command', is_flag=True, help='only show the MIP command')
+@click.option('--startfrom', help='start the pipeline from this step, see format for program using mip --help')
 @click.argument('family', required=False)
 @click.pass_context
-def start(context, mip_config, email, priority, dryrun, command, family):
+def start(context, mip_config, email, priority, dryrun, command, startfrom, family):
```

Comment:
Should you not call it "start_with_program" to mimic MIP syntax? I might add a start_from_program flag later and they imply different starting points.

Reply:
kindof weird comment. This is a wrapper, which follows the click and python syntax. All options in TB and CG are very very consice (and use `-` as delimiter ;))